data_steward/analytics/tools/dq_issue_tracker/general_functions.py

```diff
@@ -45,24 +45,24 @@ def load_files(sheet_name, file_name):
         sheet = pd.read_excel(file_name, sheet_name=sheet_name)
 
         if sheet.empty:
-            print("WARNING: No data found in the {} sheet "
-                  "in dataframe {}".format(
-                   sheet_name, file_name))
+            print(f"""WARNING: No data found in the {sheet_name} sheet
+                  in dataframe {file_name}""")
 
     except Exception as ex:  # sheet not in specified excel file
         if type(ex).__name__ == "FileNotFoundError":
```

Comment:
You could use this instead
```suggestion
    except FileNotFoundError as ex:  # sheet not in specified excel file
```
This will run into other errors and fail, but the behavior will be the same as using `sys.exit(0)`, without the print statement.
Using `Exception` is not typically recommended since it is a blanket term and will not allow the script to fail when it should, potentially causing issues downstream. I'm guessing that's why the script is set to exit with `sys.exit(0)`.

Also, this is a job for Python's `logging` module. You can set a logger to do this, and call `logger.exception(msg)` where the message is your print statement. For other Exceptions, the program should be allowed to fail.

All of the print messages should ideally be handled by a logging module with different levels, like `info`, `warning`, `error` etc., but this can be a future improvement. You might want to add this to the ticket.